### ISSUE TITLE
The loss is nan when I train the train_ccm.py

### ISSUE THREAD

**Issue by xixixihean** [YeahHighly/TIP-2024-VSR-Net#2]
Hello, I used the same dataset and parameter settings as yours, but the training loss immediately becomes 'nan'. Even after changing the dataset or adjusting the parameters, the loss remains 'nan'. Have you encountered this issue before?

**Comment by YeahHighly** [comment]
It might be due to the lack of a fixed random seed, which can sometimes lead to the loss staying as NaN. You could try tuning the hyperparameters to debug the issue. I'm sorry, but as my PhD research has shifted to a new field outside of computer vision, I've become very busy and may no longer have the time or energy to continue maintaining this project.
